pytest_ordering/__init__.py

```diff
@@ -80,3 +95,22 @@ def pytest_collection_modifyitems(session, config, items):
     sorted_items.extend([i[1] for i in end_list])
 
     items[:] = [item for sublist in sorted_items for item in sublist]
+    
+    def _get_item_index_by_name(item_name):
+        index = None
+        for i, item in enumerate(items):
+            if getattr(item, "name") == item_name:
+                index = i
+                break
+        return index
+        
+    for before_item_relative, _before_items in before_items.items():
+        index = _get_item_index_by_name(before_item_relative)
```

Comment:
Note that for invalid test names in `before` or `after` the test is just skipped - would probably be better to issue a warning and ignore the marker

Reply:
Fixed now. Also, tests for warnings added.